admin/dt-prayer-campaigns.php

```diff
@@ -161,12 +161,15 @@ public function content() {
                         <button
                             class="button"
                             id="campaign-wizard-toggle"
-                            style="display: <?php echo esc_attr( $is_wizard_open ? 'none' : 'block' ) ?>; margin-bottom: 1rem;"
+                            style="display: <?php echo esc_attr( $is_wizard_open ? 'none' : 'inline-block' ) ?>; margin-bottom: 1rem;"
                         >
-                            Campaign Wizard
+                            Show Campaign Wizard
                         </button>
 
-                            <div id="campaign-wizard" style="display: <?php echo esc_attr( $is_wizard_open ? 'block' : 'none' ) ?>;">
+                        <a class="button" href="<?php echo esc_html( home_url() ); ?>">Go to Landing Page</a>
```

Comment:
would it be nicer for this to be a target="_blank" one as well?

Reply:
Done